client/my-sites/plugins/categories/style.scss

```diff
@@ -0,0 +1,37 @@
+.categories__discover {
+	.categories__header {
+		@extend .wp-brand-font;
+		font-size: $font-title-large;
+		display: flex;
+		align-items: center;
+		cursor: pointer;
+	}
+
+	.categories__select-list {
+		background: #656667;
+		border: 1px solid #000000;
+		box-sizing: border-box;
+		border-radius: 15px; /* stylelint-disable-line */
```

Comment:
any reason for this disable?

Reply:
I think this was from @lupus2k we can remove it in a follow up if it's an issue. I suspect there are some border-radius variables we should be using instead.

Reply:
> I suspect there are some border-radius variables we should be using instead.

I think the same